config/pipeline.yaml

```
@@ -1913,6 +1934,14 @@ build_configs:
     branch: 'for-linus'
     variants: *build-variants
 
+  clk:
+    tree: clk
+    branch: 'clk-next'
+
```

```suggestion
    variants: *build-variants

```

For consistency (to be added to other build config entries as well).

---

@a-wai I've not been specifying variants for newly added trees from the time when it was indicated that these variants are legacy way of specifying them and we should avoid them. Why should variant be specified for these trees?